src/docs/manual/mockJOptionPane.java.txt

```diff
@@ -1,10 +1,8 @@
-package com.mypackage;
+package projectmock.javax.swing;
```

Comment:
I put a link to a mock file.  The link has prefix `randoop/randoop/blob/master/`, and will only work once the branch is merged.